Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN apk add --no-cache \
   git \
   tini
 
+RUN npm install --global npm npm-check-updates
```

Comment:
(issue): I'm a bit bothered by using `npm-check-updates`. From what I can read it `upgrades your package.json dependencies to the latest versions, ignoring specified versions.`, which seems to be a dangerous thing to do? 

Also, it doesn't install nor update the `package.lock`, so I'm not sure to understand why we have it here in the Dockerfile,  as we only aim to install dependencies here, not update them.

Reply:
I'm using it with a `make shell` command, and then running `ncu -u` inside the context of the container.

As the `package*json` files (normal and lock) are bind-mounted, it allow me to update it interactively (and test things).

`ncu` is not executed automatically of course :)

Reply:
Ah didn't quite catch that.  Thanks for clarifying. 

(I'll  still mumble about ignoring versions but I guess it is how it is ^^)